index.js

```diff
@@ -1,18 +1,58 @@
 /***** Deliverable 1 *****/
 const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+// console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+let headerElement = document.querySelector('#header')
+headerElement.style.color = 'blue'
```

Comment:
Your code works fine, but there's no need to create a new variable here - you can just use the `header` variable from deliverable 1!